compiler/testData/ir/irText/classes/implicitNotNullOnDelegatedImplementation.fir.txt

```diff
@@ -22,9 +22,9 @@ FILE fqName:<root> fileName:/implicitNotNullOnDelegatedImplementation.kt
       BLOCK_BODY
         DELEGATING_CONSTRUCTOR_CALL 'public constructor <init> () [primary] declared in <root>.JFoo'
         INSTANCE_INITIALIZER_CALL classDescriptor='CLASS CLASS name:K1 modality:FINAL visibility:public superTypes:[<root>.JFoo]'
-    FUN FAKE_OVERRIDE name:foo visibility:public modality:OPEN <> ($this:<root>.JFoo) returnType:kotlin.String [fake_override]
+    FUN FAKE_OVERRIDE name:foo visibility:public modality:OPEN <> ($this:<root>.JFoo) returnType:@[FlexibleNullability] kotlin.String [fake_override]
```

Comment:
As shown here: https://github.com/JetBrains/kotlin/blob/master/compiler/testData/ir/irText/classes/implicitNotNullOnDelegatedImplementation.kt#L19 nullability assertions are expected at some places, and indeed added here, but with different attribute: https://github.com/JetBrains/kotlin/blob/master/compiler/testData/ir/irText/classes/implicitNotNullOnDelegatedImplementation.txt#L30

Not so sure how it's the other way around... need to study https://github.com/JetBrains/kotlin/commit/8cb8284957f864615c81de8aaaa57781aec52afd :\

Reply:
After changing the whole logic (from enhancement point to flexible type creation point), this is no longer the case. I think there is an override checker in `java` module, and that's where to propagate attributes on the return type of overridden functions to overriding functions.